### DISCUSSION TITLE
List of sites still using v1

### DISCUSSION THREAD

**Discussion by lisa761** [facebook/docusaurus#6999] [v1-v2 migration]
List of sites that are still using Docusaurus v1:
- [ ] [Ax](https://ax.dev/) 
- [ ] [Bowler](https://pybowler.io/)
- [ ] [FastText](https://fasttext.cc)
- [ ] [Instant Articles Builder](https://facebook.github.io/instant-articles-builder/)
- [x] [Jest](https://jestjs.io)
- [ ] [MakeItOpen](https://makeitopen.com/)
- [ ] [Netconsd](https://facebookmicrosites.github.io/netconsd/)
- [ ] [React 360](https://facebook.github.io/react-360)
- [ ] [Reason](https://reasonml.github.io/)
- [ ] [ReasonReact](https://reasonml.github.io/reason-react/)
- [ ] [Redex](https://fbredex.com/)
- [x] [Relay](https://facebook.github.io/relay/)
- [ ] [Skip](http://skiplang.com)
- [ ] [Spectrum](https://libspectrum.io)
- [ ] [TorchCraftAI](https://torchcraft.github.io/TorchCraftAI/)
- [ ] [1Hive](https://1hive.org)
- [ ] [Accord Project](https://docs.accordproject.org/)
- [ ] [Adapt.js](https://adaptjs.org/)
- [ ] [All Contributors](https://allcontributors.org/)
- [ ] [Almin](https://almin.js.org/)
- [ ] [Almond](https://almond.sh/)
- [ ] [Anssr Data Platform](https://anssr.io/) (website not available)
- [ ] [Apify SDK](https://sdk.apify.com/)
- [ ] [Ara Framework](https://ara-framework.github.io/website/)
- [ ] [Babel](https://babeljs.io/)
- [ ] [Bemuse](https://bemuse.ninja/project/)
- [ ] [Bitcoin-S](https://bitcoin-s.org)
- [ ] [Bottender](https://bottender.js.org/)
- [ ] [CapRover](https://caprover.com/)
- [ ] [Casbin](https://casbin.org/)
- [ ] [Channelize.io](https://docs.channelize.io/)
- [ ] [CodeceptJS](https://codecept.io/)
- [ ] [Cohesity](https://developer.cohesity.com)
- [ ] [Collective](https://collective.gg/)
- [ ] [Day.js](https://day.js.org/)
- [ ] [Dep](https://golang.github.io/dep/)
- [ ] [DevSpace](https://devspace.cloud/docs/)
- [ ] [Dime](https://docs.dimescheduler.com)
- [ ] [Draftail](https://www.draftail.org/)
- [ ] [Express Validator](https://express-validator.github.io/)
- [ ] [FuseBox](https://fuse-box.org)
- [ ] [Global CTO Forum](https://globalctoforum.org/)
- [ ] [Goby](https://goby-lang.org) (website not available)
- [ ] [graphql-compose](https://graphql-compose.github.io)
- [ ] [Haul](https://callstack.github.io/haul/)
- [ ] [Hazelcast Jet](https://jet-start.sh)
- [ ] [Hemera](https://hemerajs.github.io/hemera/)
- [ ] [Jafar](https://yahoo.github.io/jafar)
- [ ] [KaTeX](https://katex.org/)
- [ ] [LisanJS](https://lisanjs.com)
- [ ] [manylesson](http://www.manylesson.com/)
- [ ] [Microkubes](https://www.microkubes.com/)
- [ ] [minapp](https://qiu8310.github.io/minapp/)
- [ ] [Mockit](https://mockit.netlify.com/)
- [ ] [molecularjs-boilerplate](https://pankod.github.io/moleculerjs-boilerplate/)
- [ ] [Mojito](https://mojito.mx/)
- [ ] [next-boilerplate](https://pankod.github.io/next-boilerplate/)
- [ ] [Noderize](https://noderize.js.org/)
- [ ] [Open Power Quality](https://openpowerquality.org/)
- [ ] [Overwolf](https://overwolf.github.io/)
- [ ] [Plus (ThinkSNS+)](https://slimkit.github.io)
- [ ] [Paynow](https://developers.paynow.co.zw/)
- [ ] [PayTabs](https://dev.paytabs.com/) (redirects to a login page)
- [x] [pnpm](https://pnpm.js.org/)
- [ ] [Polymath Network](https://polymath.network)
- [ ] [Prettier](https://www.prettier.io)
- [ ] [Pulsar](https://pulsar.apache.org/)
- [ ] [Quix](https://wix.github.io/quix/)
- [ ] [Quicksi](https://quicksi-bot.now.sh/)
- [ ] [Rawsec Inventory](https://inventory.rawsec.ml/docs/)
- [ ] [React Native Sensors](https://react-native-sensors.github.io/)
- [ ] [react-slate](https://react-slate.surge.sh/)
- [ ] [reaction-commerce](https://docs.reactioncommerce.com/)
- [ ] [Reactive Interaction Gateway](https://accenture.github.io/reactive-interaction-gateway/)
- [ ] [Rejoiner](https://rejoiner.io/)
- [ ] [rest-hapi](https://jkheadley.github.io/rest-hapi/)
- [ ] [Scalafmt](https://scalameta.org/scalafmt/)
- [x] [ScanTrust](https://devportal.scantrust.com/)
- [ ] [Shrine](https://shrinerb.com)
- [ ] [Stackery](https://docs.stackery.io)
- [ ] [StreamPipes](https://docs.streampipes.org) (website not available)
- [ ] [textlint](https://textlint.github.io/)
- [ ] [tipsi-stripe](https://tipsi.github.io/tipsi-stripe/)
- [ ] [TurtleCoin Docs](https://docs.turtlecoin.lol)
- [ ] [Twirp](https://twitchtv.github.io/twirp/)
- [ ] [TypeGraphQL](https://typegraphql.ml/)
- [ ] [Type Route](https://www.type-route.org)
- [x] [Verdaccio](https://www.verdaccio.org/)
- [ ] [Vuls](https://vuls.io/)
- [ ] [WarriorJS](https://warrior.js.org)

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?
Yes

**Comment by slorber** [comment]
Thanks @lisa761 , that will be very helpful :)
